tests/BaGetter.Tests/NugetBasicAuthenticationHandlerTests.cs

```diff
@@ -0,0 +1,178 @@
+using System;
```

Comment:
I think both the handler and this test would be better positioned in `BaGetter.Web` and `BaGetter.Web.Tests`, as the `BaGetter` project should mostly be configuration and tying things together. In `BaGetter.Tests` I'd like an integration tests that spins up the `BaGetter` application, if possible. If that's too much for now don't worry about the integration tests though, I can do that too eventually (i.e. when my bachelor's thesis is done).

Reply:
Plus I gotta look into that later but I think the handler being in `.Web` will also pave the way to guarding the pages with basic authentication too, if configured. I will learn about that eventually.

Reply:
The integration tests are in there already. I did move the handler and handler test as requested